pkg/core/mpt/trie.go

```diff
@@ -402,9 +402,9 @@ func makeStorageKey(mptKey util.Uint256) []byte {
 	return append([]byte{byte(storage.DataMPT)}, mptKey[:]...)
 }
 
-// Flush puts every node in the trie except Hash ones to the storage.
-// Because we care only about block-level changes, there is no need to put every
-// new node to storage. Normally, flush should be called with every StateRoot persist, i.e.
+// Flush puts every node in the trie except Hash ones, putting then to the storage.
```

Comment:
`Put` is mentioned two times.

Reply:
Changed `putting` to `adding`.

Reply:
Not exactly. The meaning is that `every node in the trie` should be put into the storage. We don't put nodes in the trie (in this context).